source/installating.rst

```diff
@@ -0,0 +1,79 @@
+.. _psp.installing:
+
+Installing |ppg|
+********************************************************************************
+
+Using repositories provided by |percona| is the recommended way of installing
+|ppg|. Installation packages are available in the |rpm| format for Red Hat,
+CentOS, and Amazon Linux AMI as well as in the |deb| format for |ubuntu| and
+|debian| distributions). |ppg| is only available for the x86_64 platform (also
+known as amd64).
+
+Like many other |percona| products, |ppg| is installed by using the
+|percona-release| utility. As soon as it is ready, enable |ppg| and then install
+using the commands of your package manager.
+
+Make sure to run |percona-release| as root or via |sudo|. For convenience, all
+commands that require elevated priviges start with |sudo|.
+
+.. code-block:: bash
+
+   $ sudo percona-release enable ppg-11
```

Comment:
In general this command will not work as it is needed to set which repo should be enabled experimental/testing/release